SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/CheckoutRadiusConstraintUtils/CheckoutRadiusConstraintUtilsTest.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
There's no need to create a folder `CheckoutRadiusConstraintUtils`. So the path should be `SpatialGDKTests/SpatialGDK/Utils/CheckoutRadiusConstraintUtilsTest.cpp`

Reply:
Resolving, since the guidelines tell to create the folder, and that's exactly what's been done here.